### PR TITLE
Fixes from m_reza_rahman

### PR DESCRIPTION
**Pull request by edburns**

### CODE REVIEW

**Review by mriccell — APPROVED** [review]

**Comment by m-reza-rahman** [comment]
I just submitted the OCA. It should be simple enough to do and I think minimizes hassle going forward.

**Comment by m-reza-rahman** [comment]
@edburns, this is now ready to move forward. I have done at least one pass over everything.

**Comment by edburns** [comment]
@galiacheng is going to push some more commits to fix the problems you identified, then we will advocate for merging this.

**Comment by galiacheng** [comment]
@edburns @rjeberhard The PR is ready for review. Please note that this PR introduces changes to the sample script. Could you please refresh the sample-scripts.zip in v4.2.8?

File: documentation/site/content/samples/azure-kubernetes-service/includes/create-aks-cluster-body-01.txt [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 
-You will need an Oracle account. The following steps will direct you to accept the license agreement for WebLogic Server.  Make note of your Oracle Account password and email.  This sample pertains to 12.2.1.4, but other versions may work as well.
+The following steps will direct you to accept the license agreement for WebLogic Server. Make note of your Oracle Account password and email. This sample pertains to 12.2.1.4, but other versions may work as well.
 
   - In a web browser, navigate to https://container-registry.oracle.com and log in using the Oracle Single Sign-On authentication service. If you do not already have SSO credentials, at the top of the page, click the **Sign In** link to create them.
   - The Oracle Container Registry provides a WebLogic 12.2.1.4 General Availability (GA) installation image that is used in this sample.
+    - In the Oracle Container Registry, navigate to **Middleware**, then **weblogic**.
+    - On the left, choose a language and accept the license agreement. You will then see a message such as: "You last accepted the Oracle Standard Terms and Restrictions on 08/10/2020 at 06:12 AM Coordinated Universal Time (UTC)."
```
Comment (edburns) [inline documentation/site/content/samples/azure-kubernetes-service/includes/create-aks-cluster-body-01.txt:8]: Thank you.

**Review by edburns — COMMENTED** [review]
Even though I am the submitter of this PR, the commits were authored by @galiacheng and @m_reza_rahman. I certify these are correct and apply my OCA to their work.

**Review by rjeberhard — APPROVED** [review]

**Review by mriccell — APPROVED** [review]

**Pull request merged**